gutenberg.php

```diff
@@ -292,16 +298,43 @@ function gutenberg_add_edit_link( $actions, $post ) {
 	return $actions;
 }
 
+/**
+ * Removes the Edit action from the reusable block list's Bulk Actions dropdown.
+ *
+ * @since 3.8.0
+ *
+ * @param array $actions Bulk actions.
+ *
+ * @return array Updated bulk actions.
+ */
+function gutenberg_block_bulk_actions( $actions ) {
+	unset( $actions['edit'] );
+	return $actions;
+}
+add_filter( 'bulk_actions-edit-wp_block', 'gutenberg_block_bulk_actions' );
+
 /**
  * Prints the JavaScript to replace the default "Add New" button.$_COOKIE
  *
  * @since 1.5.0
  */
 function gutenberg_replace_default_add_new_button() {
 	global $typenow;
+
+	if ( 'wp_block' === $typenow ) {
+		?>
+		<style type="text/css">
+			.page-title-action {
+				display: none;
```

Comment:
Was chatting to @azaozz about this. In the medium term, we should patch Core to add a filter here:

https://github.com/WordPress/WordPress/blob/master/wp-admin/edit.php#L352

This would allow us to remove this hacky way of hiding the _Add New_ button and improve how `gutenberg_replace_default_add_new_button` works.